tests/ui/conftest.py

```diff
@@ -19,13 +14,14 @@ def firefox_options(firefox_options):
 
     """
     firefox_options.set_preference(
-        'extensions.install.requireBuiltInCerts', False)
-    firefox_options.set_preference('xpinstall.signatures.required', False)
-    firefox_options.set_preference('extensions.webapi.testing', True)
-    firefox_options.set_preference('ui.popup.disable_autohide', True)
-    firefox_options.add_argument('-foreground')
-    firefox_options.add_argument('-headless')
-    firefox_options.log.level = 'trace'
+        "extensions.install.requireBuiltInCerts", False
+    )
+    firefox_options.set_preference("xpinstall.signatures.required", False)
+    firefox_options.set_preference("extensions.webapi.testing", True)
+    firefox_options.set_preference("ui.popup.disable_autohide", True)
+    firefox_options.add_argument("-foreground")
+    firefox_options.add_argument("-headless")
+    firefox_options.log.level = "trace"
```

Comment:
We are using single quotes throughout our whole code, let's not change this needlessly.